data/guide.xml

```diff
@@ -22,7 +22,10 @@
 		<p>The Weather Channel<br/>Ch. 95</p>
 	</ad>
 </ads>
-<channel number="1" name="KCBS">
+<channel number="2" name="HTLNFO">
+	<listing timeslot="1" type="1" data2="0" data3="0" data4="0" continuous="continuous">Hotel Information</listing>
```

Comment:
@alexdresko so this file looks a bit funky right now because I'm trying to reverse engineer data from the original program guide (which was a binary `.dat` blob). From what I can tell, the first data block describes the type of program. In the guide data right now you'll see some `type="movie"` entries, which I deduced since all movies had the same `data1` entry, and no programs that weren't movies had it. I'm not sure what the others were intended to store yet, but it might be better to remove them all for now.